### PR TITLE
openbb_terminal tests: coverage (61% -> 65%)

### LINKED ISSUE #1667: [Bug] Econometrics Export FileNotFound
**Describe the bug**
Export in Econometrics causes FileNotFoundError.

**To Reproduce**
`load cancer dataset`
`export dataset`

**Screenshots**
Screen Shot 2022-04-10 at 1 47 13 PM

**Desktop (please complete the following information):**

- OS: Mac Monterey
- Python version 3.8

### PR DESCRIPTION
**Pull request by colin99d**
# Description

Expanding general terminal tests. Will plan on being ready to merge on Sunday night regardless of how many tests have been written: Adding tests to:

- Keys Controller
- Econometrics Controller
- Settings Controller
- common/qa_view

@JerBouma I also added some enhancements to the econometrics controller to make it more robust. Can you please confirm these changes are desired?

@Chavithra the tests pass now. I BELIEVE `matplotlib.pyplot.show` is now fully mocked.

Also, Fixes #1667 

# How has this been tested?

Ran pytest.

# Checklist:

- [x] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [x] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

**Comment by colin99d** [comment]
@Chavithra I cannot figure out how to mock matplotlib so that it will not show graphs in `tests/openbb_terminal/common/quantitative_analysis/test_qa_view.py`. Any advice?

**Comment by JerBouma** [comment]
@colin99d Looks good to me! 👍

File: tests/openbb_terminal/econometrics/txt/test_econometrics_controller/test_call_bgod[other1].txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+The following args couldn't be interpreted: ['-n', 'dataset']
```
Comment (Chavithra) [inline tests/openbb_terminal/econometrics/txt/test_econometrics_controller/test_call_bgod[other1].txt:2]: Is there an issue there ?

**Review by Chavithra — COMMENTED** [review]
There is an issue on my Windows :
```
>           path_cmd = dir_path.split("openbb_terminal/")[1].replace("/", "_")
E           IndexError: list index out of range
```

Maybe that's related to the OS separator.

```

========================================================================================================================== short test summary info ========================================================================================================================== 
FAILED tests/openbb_terminal/common/quantitative_analysis/test_qa_view.py::test_display_summary - IndexError: list index out of range
FAILED tests/openbb_terminal/common/quantitative_analysis/test_qa_view.py::test_display_cdf[None] - IndexError: list index out of range
FAILED tests/openbb_terminal/common/quantitative_analysis/test_qa_view.py::test_display_seasonal[False] - IndexError: list index out of range
FAILED tests/openbb_terminal/common/quantitative_analysis/test_qa_view.py::test_display_seasonal[True] - IndexError: list index out of range
FAILED tests/openbb_terminal/common/quantitative_analysis/test_qa_view.py::test_display_normality - IndexError: list index out of range
======================================================================================================================= 5 failed, 23 passed in 5.71s ========================================================================================================================
```

File: tests/openbb_terminal/econometrics/txt/test_econometrics_controller/test_call_bgod[other1].txt (hunk 1 above)
Comment (tehcoderer) [reply tests/openbb_terminal/econometrics/txt/test_econometrics_controller/test_call_bgod[other1].txt:2]: @Chavithra can you try with my changes and let me know if that resolves it.. haha get it? I'll walk myself out...

Comment (colin99d) [reply tests/openbb_terminal/econometrics/txt/test_econometrics_controller/test_call_bgod[other1].txt:2]: @Chavithra this is not an issue.

**Review by tehcoderer — APPROVED** [review]

**Pull request merged**